libcaf_test/caf/test/runner.cpp

```diff
@@ -68,7 +68,17 @@ int runner::run(int argc, char** argv) {
     return EXIT_SUCCESS;
   }
   default_reporter->start();
+  auto enabled = [](const std::regex& selected,
+                    const std::string& search_string) {
+    return std::regex_search(search_string, selected);
+  };
```

Comment:
While `regex` doesn't support `string_view` directly, you can still use string views and pass in an iterator pair: https://en.cppreference.com/w/cpp/regex/regex_search.

```suggestion
  auto enabled = [](const std::regex& selected,
                    std::string_view search_string) {
    return std::regex_search(search_string.begin(), search_string.end(), selected);
  };
```